priv/xcodegen/Sources/TemplateApp/ContentView.swift

```diff
@@ -15,78 +15,14 @@ struct ContentView: View {
             ),
             addons: []
         ) {
+          // connecting
```

Comment:
because Swift doesn't support labels for the default closure I think this comment at least implies its purpose for those that are not familiar with Swift